packages/ui/Scrollable/ScrollableNative.js

```diff
@@ -759,79 +769,74 @@ class ScrollableBaseNative extends Component {
 		return clamp(0, 1, 2 * overDistance / baseSize);
```

Comment:
It would be better to add the following `else` cluase to reduce unnecessary calculation.
```
	if (position < 0) {
		overDistance = -position;
	} else if (position > maxPos) {
		overDistance = position - maxPos;
	} else {
		return 0;
	}
```

Reply:
Simply, use Math.min instead of clamp for this case.
```
Math.min(1, 2 * overDistance / baseSize);
```

Reply:
Okay!